src/main/resources/jenkins/plugins/openstack/compute/SlaveOptions/config.jelly

```diff
@@ -17,6 +17,12 @@
         <div name="slaveOptions" descriptorid="${descriptor.id}">
             <table style="width:100%">
                 <f:section title="Cloud Server Options">
+                    <f:entry title="Boot Source" field="bootSource">
+                        <f:select onchange="this.parentElement.parentElement.parentElement.querySelector('[invisibleCopyOf]').value = this.value" />
```

Comment:
**So would I !**
If you know of a way around this then I'll give it a try.  I think I tried just about every other possibility before I resorted to this method.

See the comment in SlaveOptionsDescriptions.java line 277 - in order to read "bootSource" from both "our slave options" and "the default slave options" we need the value of the "bootSource" field to be readable by two differently-named fields, otherwise the Jenkins<->Jelly bindings goes wrong and passes the same value back to both parameters in the doFill and doCheck methods.

i.e. There's a Jenkins bug/"feature" and this is the workaround.